src/__swaps__/screens/Swap/resources/assets/userAssets.ts

```diff
@@ -124,12 +121,14 @@ async function userAssetsQueryFunction({
     const chainIdsWithErrorsInResponse = res?.data?.meta?.chain_ids_with_errors || [];
     const assets = res?.data?.payload?.assets || [];
     if (address) {
-      userAssetsQueryFunctionRetryByChain({
-        address,
-        chainIds: chainIdsWithErrorsInResponse,
-        currency,
-        testnetMode,
-      });
+      if (chainIdsWithErrorsInResponse.length) {
```

Comment:
I'm only calling the `userAssetsQueryFunctionRetryByChain` when there are `chainIdsWithErrorsInResponse` (this is mostly a no-op but we do avoid doing unnecessary function calls here)